test/k8sT/Services.go

```diff
@@ -2080,6 +2080,13 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 				testExternalTrafficPolicyLocal()
 			})
 
+			It("with reverse NAT at host netfilterCompatMode=true", func() {
```

Comment:
This case is not needed, as this case runs only with kube-proxy and without the kube-proxy-replacement. Or is it intended? If yes, then this needs a comment.